pkg/project/registry/project/proxy/proxy.go

```diff
@@ -73,10 +72,7 @@ func convertProject(project *api.Project) *kapi.Namespace {
 			Phase: project.Status.Phase,
 		},
 	}
-	if namespace.Annotations == nil {
-		namespace.Annotations = map[string]string{}
-	}
-	namespace.Annotations["displayname"] = project.DisplayName
```

Comment:
I see here we were sticking the key as displayname in the past and not as displayName like you do in this PR.

I think it's fine to change that to use camel case (which I prefer for readability), but I need to check if that will affect @abonas since she may have been consuming the old annotation key.  We make no claim today about preserving data, so it should not be an issue to change if she has proper notice.

Aside from that, can you add a constant to pkg/project/api/types.go for display name and description, update it to be used in all places, and then we can be sure we do not have displayName and displayname discrepancies in the future.